CMake/cdat_modules/x264_deps.cmake

```diff
@@ -1 +1 @@
-set(X264_DEPS "")
+set(X264_deps ${pkgconfig_pkg})
```

Comment:
Nope; in fact, it technically doesn't even depend on `pkgconfig`. It has a soft dependency on `yasm`, but that's already sort of a dependency for us (as part of the original x264 PR I did some stuff to make our build handle not having `yasm` installed).